python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import glob
+import sys
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
It's important to separate the imports in groups: core, others, local imports. So I will leave a blank line between `sys` and `mahotas`

Reply:
I'm not sure I understand what you mean here, but I will make the changes requested

Reply:
```
import sys

import numpy as np
import matplotlib.pyplot as plt
import mahotas
```